### DISCUSSION TITLE
Single Image to 3D

### DISCUSSION THREAD

**Discussion by georgeasrian** [alicevision/Meshroom#2323] [General]
Is it possible to use a single image as the input and render a 3D model? When trying to do so with out-of-the-box settings, the workflow breaks down early on.

**Comment by natowi** [comment]
No, photogrammetry relies on at least two images.

What you are looking for is something like https://github.com/junshutang/Make-It-3D
There are other open source projects out there, some with google colab notebook you can run (https://github.com/facebookresearch/pifuhd)

Here a ready to use commercial offering: https://www.alpha3d.io/
